RPIPs/RPIP-49.md

```diff
@@ -137,10 +145,10 @@ The below is generally agreed to be the steps to be completed before we can cons
 3. **Done** - Create high-level explanations and informational material for the full proposal for consumption by the wider community.
 4. **Done** - Make a concerted effort to gather feedback via the forum from the wider community.
 5. **Active** - Update the proposal and specifications as needed taking into account wider community feedback.
-6. Run a forum temperature check vote on the full proposal (bar value capture mechanism).
-7. Run a snapshot vote on the full proposal (bar value capture mechanism).
-8. Run a snapshot vote on the value capture mechanism once outstanding blockers are cleared.
-9. Update the proposal to incorporate the value capture mechanism vote result.
+6. Run a forum temperature check vote on the rework package (bar the 'still-to-ratify' list above).
+7. Run a snapshot vote oon the rework package (bar the 'still-to-ratify' list above).
```

Comment:
```suggestion
7. Run a snapshot vote on the rework package (bar the 'still-to-ratify' list above).
```

Reply:
Sigh, should have run grammarly. Thanks Samus.